style.css

```diff
@@ -558,3 +572,438 @@ ul li {
 .about-btn:hover {
   background-color: rgb(84, 236, 173);
 }
+
+.contact-box {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 355px;
+  height: 363px;
+  justify-content: center;
+  margin-bottom: 58px;
+  text-align: center;
+}
+
+.contact-text {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: rgb(23, 43, 77);
+  align-items: center;
+  justify-content: center;
+  padding: 0 24px;
```

Comment:
- [OPTIONAL] Kindly remove the margin-top so that  you can get the same design as Figma